api/PaymentResponse.json

```diff
@@ -66,12 +89,37 @@
               "version_added": "15"
             },
             "firefox": {
-              "version_added": false,
-              "notes": "Available only in nightly builds."
+              "version_added": "55",
+              "notes": "Available only in nightly builds.",
```

Comment:
It's been a while since I've looked at this, but I _think_ we talked about dropping these notes on the suspicion that they're not accurate (otherwise, we could use `"preview"` values, if they're true).

Reply:
Whoops!  Somehow I didn't catch these notes in the removal, fixed!